docs/modules/ROOT/pages/tutorials/monitoring/index.adoc

```diff
@@ -0,0 +1,158 @@
+.. The Pushover user key and API keys, as shown on the Pushover user interface.
++
+[source]
+--
+APPUIO_CLOUD_ZONE=cloudscale-lpg-2     (or exoscale-ch-gva-2-0)
+APP_PROJECT=xxxxx-fortune
+GRAFANA_PROJECT=xxxxx-grafana
+PUSHOVER_USER_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+PUSHOVER_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+--
+
+== Deploy the Fortune in Go Application
+
+The first step is to deploy the Fortune in Go application on its APPUiO Cloud project. To make this happen, we require three familiar objects: a `Deployment`, a `Service`, and an `Ingress` object. The Deployment specifies the container to use: `registry.gitlab.com/vshn/applications/fortune-go:latest`.
+
+. Execute the following commands:
++
+[source,bash]
+--
+source variables
+oc new-project $APP_PROJECT
+oc process -f 1_deployment.yaml --param-file=variables | oc apply -f -
+--
+
+. After a few seconds, you should be able to open the corresponding route and interact with the application.
+. On the APPUiO Cloud console, select the "Developer" perspective, click on the "Observe" menu on the left, and then select the "Alerts" tab. This screen will display the alerts we will generate in the following step.
+
+== Generate and Receive Alerts
+
+After deploying the application, we will configure AlertManager and Prometheus to consume the usage information reported by the application. Doing so requires the following objects:
+
+* A `ServiceMonitor` object;
+* A `PrometheusRule` object specifying the expression to be used as a rule to determine whether an alert should be raised. In our case, we will be using this rule: `rate(fortune_generated_message[1m]) > 5`
+* And an `AlertmanagerConfig` object. This last object requires two Secret objects, specifying Pushover's "user key" and "API key."
+
+NOTE: `AlertmanagerConfig` objects can be configured with a variety of services such as Telegram, Slack, and others. The xref:how-to/configure-alertmanager.adoc[AlertManager configuration page] provides more information.
+
+Follow these steps:
+
+. Execute the following command:
++
+[source,bash]
+--
+oc process -f 2_alerts.yaml --param-file=variables | oc apply -f -
+--
+
+. Open the Fortune in Go application on your browser.
+. Press the kbd:[R] key for about 20 or 30 seconds while using the Fortune in Go application; your mobile or web Pushover clients should receive a notification in a minute or two.
+. During the interaction with the application, the Alert screen on the APPUiO Cloud console will display a message indicating that the alert has been triggered.
++
+image::monitoring/tutorial/console-monitoring-alerts-firing.png[]
+
+. The Pushover interface should show that an alert arrived, and the Pushover client should show the notification, including the error message specified in the YAML file.
++
+image::monitoring/tutorial/pushover-view-application.png[]
++
+image::monitoring/tutorial/pushover-web-client.png[]
+
```

Comment:
Please add a link to the troubleshooting section at https://docs.appuio.cloud/user/how-to/configure-alertmanager.html#troubleshooting.

Reply:
Done